src/lib/y2configuration_management/salt/form_data.rb

```diff
@@ -64,11 +56,28 @@ def update(path, value)
         parent[parts.last] = value
       end
 
+      # Adds an element to a collection
+      #
+      # @param path  [String] Path to the collection
+      # @param value [Hash] Value to add
+      def add_item(path, value)
+        collection = get(path)
+        collection.push(value)
+      end
+
+      # @param path  [String]  Path to the collection
+      # @param index [Integer] Position of the element to remove
+      # @param value [Object] New value
+      def update_item(path, index, value)
+        collection = get(path)
+        collection[index, 1] = [value]
```

Comment:
```suggestion
        collection[index] = value
```
or am I missing something?

Reply:
Oh, a leftover from previous tests...